pages/apis/rest_api/pipelines.md

```diff
@@ -876,7 +876,7 @@ Optional [request body properties](/docs/api#request-body-properties):
   </tr>
   <tr>
     <th><code>name</code></th>
-    <td>The name of the pipeline.<p class="Docs__api-param-eg"><em>Example:</em> <code>"New Pipeline"</code></p></td>
+    <td>The name of the pipeline. If you provide a new name without a <code>slug</code> parameter, the slug will be automatically updated to match the new name.<p class="Docs__api-param-eg"><em>Example:</em> <code>"New Pipeline"</code></p></td>
```

Comment:
It seems to be inconsistent whether or not we wrap these cells in a paragraph, I suggest we do so with these changes:
```suggestion
    <td>
      <p>The name of the pipeline. If you provide a new name without a <code>slug</code> parameter, the slug will be automatically updated to match the new name.</p>
      <p class="Docs__api-param-eg"><em>Example:</em> <code>"New Pipeline"</code></p>
    </td>
```